client/layout/logged-out.jsx

```diff
@@ -142,19 +145,29 @@ const LayoutLoggedOut = ( {
 
 			{ sectionName === 'plugins' && (
 				<>
-					<UniversalNavbarFooter />
-					<UniversalNavbarFooterAutomattic />
+					<UniversalNavbarFooter
+						currentRoute={ currentRoute }
+						isLoggedIn={ isLoggedIn }
+						onLanguageChange={ ( e ) => {
+							navigate( e.target.value );
+							window.location.reload();
+						} }
+					/>
 					{ config.isEnabled( 'layout/support-article-dialog' ) && (
```

Comment:
I think this async load is for the Do Not Sell my data dialog box which has been removed. Since we will be adding it again we can probably leave this here. I will get a PR submitted immediately after this one.